src/app/views/vehicle-list/vehicle-list.component.ts

```diff
@@ -7,17 +7,44 @@ import { VehicleService } from 'src/app/services/vehicle.service';
   templateUrl: './vehicle-list.component.html',
   styleUrls: ['./vehicle-list.component.css']
 })
-export class VehicleListComponent implements OnInit{
-  vehicles: Vehicle[] = [];
-  
+export class VehicleListComponent implements OnInit {
+  vehicles: Vehicle[] = [
+    {brand: 'Audi', status: 'active', transmissionType: 'manual', id: 1, yearOfFabrication: '2000'}
+  ];
+  first: number = 0;
+  rows: number = 10;
 
-  constructor(private vehicleService: VehicleService){  }
+  constructor(private vehicleService: VehicleService) { }
 
   ngOnInit(): void {
-    this.vehicleService.getVehicles().subscribe(data =>{
-      console.log(data)
+    this.vehicleService.getVehicles().subscribe(data => {
+      console.log(data);
       this.vehicles = data;
-      console.log(this.vehicles)
+      console.log(this.vehicles);
     });
   }
+
+  onPageChange(event: { first: number; rows: number; }) {
+    this.first = event.first;
+    this.rows = event.rows;
+  }
+
+  reload(): void {
```

Comment:
to nie jest dobre podejście z tym wczytywaniem od nowa całej listy z bazy danych, lepiej lokalnie skasować ten usunięty samochód